### ISSUE TITLE
Error when trying to update

### ISSUE THREAD

**Issue by dylexrocks** [hklages/node-red-contrib-sonos-plus#211]
### Description
Hi, when I try to update the package through the latest version using the node red ui, I get the below error. Unfortunately I do not have much knowledge of nodejs development, so I am stuck and would really appreciate any help pushing me in the right direction. 

```
-----------------------------------------------------------
2022-04-04T12:37:53.752Z Install : node-red-contrib-sonos-plus 6.3.0

2022-04-04T12:37:53.859Z npm install --no-audit --no-update-notifier --no-fund --save --save-prefix=~ --production --engine-strict node-red-contrib-sonos-plus@6.3.0
2022-04-04T12:37:55.918Z [err] npm
2022-04-04T12:37:55.919Z [err]  ERR!
2022-04-04T12:37:55.919Z [err]  code EINVALIDTAGNAME
2022-04-04T12:37:55.928Z [err] npm 
2022-04-04T12:37:55.928Z [err] ERR!
2022-04-04T12:37:55.928Z [err]  Invalid tag name ""~"1.0.2": Tags may not have any characters that encodeURIComponent encodes.
2022-04-04T12:37:55.953Z [err] 
2022-04-04T12:37:55.953Z [err] npm 
2022-04-04T12:37:55.953Z [err] ERR!
2022-04-04T12:37:55.954Z [err]  A complete log of this run can be found in:
2022-04-04T12:37:55.954Z [err] npm ERR!     /home/pi/.npm/_logs/2022-04-04T12_37_55_930Z-debug.log
2022-04-04T12:37:55.965Z rc=1
```
### Versions and Infrastructure

**what system:** Raspberry Pi 4 with Raspbian GNU/Linux 10 (buster)
**node-red-contrib-sonos-plus version:** v4.0.0
**Node-RED version:** v2.2.2
**NodeJS version:** v16.13.1
**NPM version:** v8.1.2

**Comment by dylexrocks** [comment]
Hi

thanks for your quick replay. Attached you find the log file.

[2022-04-04T12_37_55_930Z-debug.log](https://github.com/hklages/node-red-contrib-sonos-plus/files/8410005/2022-04-04T12_37_55_930Z-debug.log)
.

**Comment by hklages** [comment]
Is node-red working with sonos-plus 4.0.0 right now? 
Did you try to update a second time?

**Comment by dylexrocks** [comment]
Yes it is working right now. I just checked to be sure.

I tried updating at least 5 times, including rebooting the pi before trying to update.

I also tried to update the package manually by running the command from the log exactly as it is and this ran without error and then restarted node-red. However when I navigate to manage palette it still shows v4.0.0. 

Am I missing a command? Is updating the package manually outside node-red even possible/intended?

**Comment by dylexrocks** [comment]
How exactly does the command look like to update manually? I tried `npm install --no-audit --no-update-notifier --no-fund --save --save-prefix=~ --production --engine-strict node-red-contrib-sonos-plus@4.5.5` and just `npm i node-red-contrib-sonos-plus@4.5.5`in both cases I got the output, that some packages where installed/removed or up to date in the latter case. However the node red UI still shows v4.0.0 as the installed version. Do I have to run another command after the installation, like `npm rebuild`?

Sorry if this is a noob question.
